front_end/core/common/ResourceType.ts

```diff
@@ -370,6 +370,14 @@ export const resourceCategories = {
   Other: new ResourceCategory(i18nLazyString(UIStrings.other), i18nLazyString(UIStrings.other)),
 };
 
+export const resourceCategoriesReactNative = {
+  XHR: new ResourceCategory(i18nLazyString(UIStrings.fetchAndXHR), i18n.i18n.lockedLazyString('Fetch/XHR')),
+  Script: new ResourceCategory(i18nLazyString(UIStrings.javascript), i18nLazyString(UIStrings.js)),
+  Image: new ResourceCategory(i18nLazyString(UIStrings.image), i18nLazyString(UIStrings.img)),
+  Media: new ResourceCategory(i18nLazyString(UIStrings.media), i18nLazyString(UIStrings.media)),
+  Other: new ResourceCategory(i18nLazyString(UIStrings.other), i18nLazyString(UIStrings.other)),
+};
+
```

Comment:
Is it just used for options under a dropdown for filtering? If so, then I don't think we should fork it, I think its fine to keep potentially multiple resource types that we don't support yet / won't have in the future, but will leave it to you